R/package.R

```diff
@@ -391,19 +391,8 @@ renv_package_built <- function(path) {
 }
 
 renv_package_checking <- function() {
-  memo(renv_package_checking_impl())
```

Comment:
This is fast enough and called in few enough places that I don't think there's an need to memoise, and removing the cache eliminates a possible failure mode.